docs/index.rst

```diff
@@ -61,3 +61,30 @@ Features
     how-to
     how-it-works
     changelog
```

Comment:
I think it is best to put the new text in its own file, say `known-limitations.rst`, and add it above `changelog` here. As it stands, it is kind lost at the bottom of the page:

https://pytest-xdist--796.org.readthedocs.build/en/796/